src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleaseFileController.cs

```diff
@@ -90,7 +91,7 @@ public async Task<ActionResult<FileInfo>> UpdateChartFile(Guid releaseId, Guid i
         [HttpPost("release/{releaseId}/ancillary")]
         [RequestSizeLimit(int.MaxValue)]
         [RequestFormLimits(ValueLengthLimit = int.MaxValue, MultipartBodyLengthLimit = int.MaxValue)]
-        public async Task<ActionResult<FileInfo>> UploadAncillary(Guid releaseId,
+        public async Task<ActionResult<AdminFileInfo>> UploadAncillary(Guid releaseId,
```

Comment:
I've created `AncillaryFileInfo` - I did this rather than create an `AncillaryFileViewModel` because the class for the file inherits from `FileInfo` on both the frontend and the backend.

I'm going to create a separate ticket to investigate the UpdateChartFile and UploadChart returning FileInfo rather than an InfographicFileInfo or void. A quick looks suggests it can just return void, but I don't want to get caught up in that with this PR. EES-2354